src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -117,7 +117,7 @@ func doMain() error {
 
 	parseFlags()
 
-	cfg, err := config.Init()
+	cfg, err := config.Init(packageVersion)
```

Comment:
Do we want to use ``version`` here instead of ``packageVersion`` -- ``packageVersion`` looks like it defaults to ``9999.99.99``, and that's to avoid auto-updating in development mode. ``version`` contains the current git revision so we'd get ``lantern-bae7606.yaml`` instead of ``lantern-9999.99.99.yaml`` which I think is more informative.